pkg/manager/meta/meta_manager.go

```diff
@@ -73,6 +73,11 @@ func (m *metaManager) Sync(tc *v1alpha1.TidbCluster) error {
 			if pvc.Spec.VolumeName == "" {
 				continue
 			}
+
+			if m.deps.PVLister == nil {
+				klog.V(4).Infof("persistent volumes lister is unavailable, skip updating meta info for %s. this may be caused by no relevant permissions", pvc.Spec.VolumeName)
```

Comment:
```suggestion
				klog.V(4).Infof("Persistent volumes lister is unavailable, skip updating meta info for %s. This may be caused by no relevant permissions", pvc.Spec.VolumeName)
```